src/psyclone/tests/domain/lfric/psyir_test.py

```diff
@@ -115,6 +81,9 @@ def test_generic_scalars(data_type, symbol, intrinsic, precision):
     lfric_datatype = data_type()
     assert lfric_datatype.intrinsic == intrinsic
     assert lfric_datatype.precision is precision
+    # precision can be set explicitely
```

Comment:
Strictely speaking this should be "explicitly" :-) and again belowe

Reply:
Oopes